### PR TITLE
SearchBox inside Menu in Mobile Version

### PR DESCRIPTION
**Pull request by CSElonewolf**
*Previously the SearchBox disappered in mobile version but now it appears inside the menu*

### CODE REVIEW

File: antora-ui-camel/src/css/header.css [hunk 1]
```diff
@@ -416,3 +416,21 @@ body {
     display: none;
   }
 }
+
+@media screen and (min-width: 610px) {
```
Comment (zregvart) [inline antora-ui-camel/src/css/header.css]: Let's not add another media breakpoint, please use the ones we have already.

**Comment by CSElonewolf** [comment]
> The checks fail due to repeating id:
> 
> ```
> [2020-03-23T17:09:51.309Z] public/acknowledgments/index.html
> [2020-03-23T17:09:51.309Z]   1:3599  error  Duplicate ID "search_mobile"          no-dup-id
> [2020-03-23T17:09:51.309Z]   1:3679  error  Duplicate ID "search_results_mobile"  no-dup-id
> [2020-03-23T17:09:51.309Z]   1:4330  error  Duplicate ID "search_mobile"          no-dup-id
> [2020-03-23T17:09:51.309Z]   1:4410  error  Duplicate ID "search_results_mobile"  no-dup-id
> [2020-03-23T17:09:51.309Z]   1:5135  error  Duplicate ID "search_mobile"          no-dup-id
> [2020-03-23T17:09:51.309Z]   1:5215  error  Duplicate ID "search_results_mobile"  no-dup-id
> ```
> 
> We also need the corresponding change in the Anotra UI layout as this changes only the Hugo layout.

I couldn't find the actual file inside [link](https://github.com/apache/camel-website/tree/master/antora-ui-camel/src/partials).
Please can you help me with this

**Comment by CSElonewolf** [comment]
@zregvart I have made the requested Changes

File: antora-ui-camel/src/partials/header-content.hbs [hunk 2]
```diff
@@ -15,6 +15,10 @@
                 {{/each}}
                 </div>
                 </div>
+                <div class="navbar-search results-hidden">
+                  <input id="search_mobile" class="search" placeholder="Search" autocomplete="off">
+                  <div id="search_results_mobile"></div>
+                </div>
```
Comment (zregvart) [inline antora-ui-camel/src/partials/header-content.hbs]: I think problem with this is that it's rendered multiple times (notice the `#each @items` on line 8.

**Comment by zregvart** [comment]
@CSElonewolf remember that you can build locally to preview the result, this is how this change currently looks like:

Screenshot_2020-03-27 Bean method Language - Apache Camel

**Comment by CSElonewolf** [comment]
Then I think we need to place it after `{{/withMenuData}}` at line 22

File: layouts/partials/header.html [hunk 3]
```diff
@@ -46,6 +46,10 @@
                                 {{ end }}
                             </div>
                         </div>
+                        <div class="navbar-search results-hidden">
+                            <input id="search_mobile" class="search" placeholder="Search" autocomplete="off">
+                            <div id="search_results_mobile"></div>
+                        </div>
```
Comment (zregvart) [inline layouts/partials/header.html]: This is also within the range loop on line 39, I think it should be placed outside as in the Antora UI.

Comment (CSElonewolf) [reply layouts/partials/header.html]: No.But I have placed it outside the as I have placed in Antora UI.
Actually I couldn't find till which line the range of the loop is in Hugo.

Comment (zregvart) [reply layouts/partials/header.html]: Odd that you say that it's not within the loop then. The range loop starts on line 39 and ends on line 56

**Comment by zregvart** [comment]
@CSElonewolf are you working on this?

**Comment by CSElonewolf** [comment]
I am currently not working on this.Actually I got little bit confused with the range of the loop and thus I am struggling with it.

**Comment by zregvart** [comment]
We're taking a different approach in CAMEL-14958, I'm going to close this.

**Pull request closed without merging**